docs/conf.py

```diff
@@ -39,6 +39,9 @@
     "linkify",
 ]
 
+# Avoid duplicate labels when multiple documents share identical headings
+autosectionlabel_prefix_document = True
```

Comment:
Have you tried running the `ddev start` or `ddev restart` again after you applied this? The duplicate labels warnings are gone, but there are new warnings appear.

Please test it out and let us know how it goes.

---

@RCheesley, do we want to fix it in this PR?

Reply:
I tried and it didn't work @adiati98. 😞

Reply:
Do you see them now @adiati98?

Reply:
Hey @RCheesley. It works now that the changes were removed